tests/br_debug_meta/run.sh

```diff
@@ -15,24 +15,23 @@
 
 set -eu
 DB="$TEST_NAME"
-TABLE="usertable1"
+TABLE="usertable"
 
 run_sql "CREATE DATABASE $DB;"
+go-ycsb load mysql -P tests/$TEST_NAME/workload -p mysql.host=$TIDB_IP -p mysql.port=$TIDB_PORT -p mysql.user=root -p mysql.db=$DB
 
-run_sql "CREATE TABLE $DB.$TABLE( \
-  YCSB_KEY varchar(64) NOT NULL, \
-  FIELD0 varchar(1) DEFAULT NULL, \
-  PRIMARY KEY (YCSB_KEY) \
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;"
-
-run_sql "INSERT INTO $DB.$TABLE VALUES (\"a\", \"b\");"
-run_sql "INSERT INTO $DB.$TABLE VALUES (\"aa\", \"b\");"
+for i in $(seq 10); do
+    regioncount=$(run_sql "SHOW TABLE $DB.$TABLE REGIONS;" | grep "REGION_ID" | wc -l)
+    [ $regioncount -ge 2 ] && break
+    sleep 3
+done
+run_sql "SHOW TABLE $DB.$TABLE REGIONS;" | grep "REGION_ID"
```

Comment:
🤔 

```sql
select count(*) from information_schema.tikv_region_status
where db_name = '$DB' and table_name = '$TABLE';
```

also you could use `split table $DB.$TABLE by (some_pk_value)` to immediately split the table into two regions, no need to do this loop.

Reply:
Could you elaborate on how to split table?  Tried `run_sql "split table $DB.$TABLE by ('user6282602628620641459', 'user6282603728132269670', 'user6282604827643897881', 'user6282610325202038936')"`, but it does not work out.

Reply:
@overvenus try `split table $DB.$TABLE by ('user6282602628620641459'), ('user6282603728132269670'), /*etc*/`;